content/en/about/contributing/git/_index.md

```diff
@@ -170,6 +173,8 @@ you can reuse the provided environment with:
 ```shell
 # From the root of the repository clone
 $ flox activate
+# Then install docsy dependencies with
+npm ci
```

Comment:
Oh, in fact I thought that the flox env would also take care of this.
Please also add the `$` marker.

Reply:
yes, sorry for not being clear; it's been added now.